client/components/mma/switch/SwitchContainer.tsx

```diff
@@ -173,6 +177,10 @@ function userIsNavigatingBackFromCompletePage(hasCompleted: boolean) {
 	return hasCompleted && !location.pathname.includes('complete');
 }
 
+function hasNonServiceableCountry(productDetail: ProductDetail) {
```

Comment:
would this also be better as `isNonServiceableCountry`? in fact it's the exact same function as above so you could pull it out to somewhere else, like in Utilities

Reply:
Moved into ProductUtils as previously discussed